assets/js/screens/Work/Work.jsx

```diff
@@ -85,6 +120,12 @@ const ScreensWork = () => {
       </section>
       <UIBreadcrumbs items={breadCrumbs} data-testid="work-breadcrumbs" />
       <Work work={data.work} />
+      <UIModalDelete
+        isOpen={deleteModalOpen}
+        handleClose={onCloseModal}
+        handleConfirm={handleDeleteClick}
+        thingToDeleteLabel={`Work ${descriptiveMetadata.title || ""}`}
```

Comment:
Can you add some type of check here in case of `descriptiveMetadata.title` doesn't exist?   Currently it says "Work null successfully deleted".  How about we not include "null" in the message?   Just "Work deleted" or something....

Reply:
I added "|| " to fix that null. Are you still getting it when deleting a work with no title?

Reply:
Could we maybe do something like show the accession number instead if there is no title? No works have titles on ingest. Works always have accession numbers...

Reply:
Yea, that might be the way to go.

Reply:
I thought about it but i expected if there's a field for title, they must be using it. I need to stop assuming :D , I'll add this and adam's suggestion as well.